bidwire/alembic/versions/9b30b0fe231a_create_documents_table.py

```diff
@@ -23,6 +23,7 @@ def upgrade():
         sa.Column('url', sa.String, unique=True),
         sa.Column('title', sa.String),
         sa.Column('site', sa.String),
+        sa.Column('description', sa.String),
```

Comment:
This is not going to work. It works in your dev environment because you are re-running all migrations every time, but if I deploy this to prod this migration won't be applied, because it has already been applied, and so your code will fail because the `description` column won't be added.

Simply create a new migration with the changes you need:
```bash
$ alembic revision -m "Add description to documents table"
```
Then in the new migration file:
```python
def upgrade():
    op.add_column('documents', sa.Column('description', sa.String))


def downgrade():
    op.drop_column('documents', sa.Column('description', sa.String))
```